### DISCUSSION TITLE
Centos7 binaries

### DISCUSSION THREAD

**Discussion by cookpa** [ANTsX/ANTs#1782] [Ideas]
Now that Centos7 is officially EOL, I'm looking at alternatives. Considering trying AlmaLinux 8.

Tagging @gdevenyi but welcome ideas from everyone else too

**Comment by gdevenyi** [comment]
The two descendents of Centos are Rocky and Alma. I don't know who is going to win in HPC long term. I'm using Ubuntu LTS for all my deployments.

**Reply by cookpa** [reply]
Thanks, Alma worked fairly painlessly so I will go with that for a RHEL-like distro for now.
